packages/authorization/src/__tests__/acceptance/authorization-casbin.acceptance.ts

```diff
@@ -26,7 +27,7 @@ describe('Authorization', () => {
   let events: string[];
 
   before(givenApplicationAndAuthorizer);
-  beforeEach(givenRequestContext);
+  beforeEach(() => givenRequestContext());
```

Comment:
Why is this change needed?

Reply:
@raymondfeng I am seeing this weird error...

![Screen Shot 2019-08-09 at 3 29 35 PM](https://user-images.githubusercontent.com/12554153/62804223-a071b280-baba-11e9-8f6b-d70cc03d0bd6.png)

Reply:
Oh, I think that's because `givenRequestContext` takes an optional `user` argument.